<launch>

	<arg name="state_inspection" default="0" />
	<arg name="parking_count" default="-3" />

	<node name="waypoint_loader_node" pkg="waypoint_maker" type="waypoint_loader" output="screen">
		<param name="state_inspection" value="$(arg state_inspection)" />
		<param name="parking_count" value="$(arg parking_count)" />
	</node>
	<node type="rviz" name="rviz" pkg="rviz" args="-d $(find waypoint_maker)/rviz/path.rviz"/>

</launch>
